src/main/java/org/moe/idea/MOESdkPlugin.java

```diff
@@ -173,31 +160,31 @@ public static boolean isValidMoeModule(Module module, boolean isMOEApp) {
         return false;
     }
 
-    public static boolean isMoeJarsInModule(Module module) {
-        if (module == null) {
-            return false;
-        }
-        if (module.isDisposed()) {
-            LOG.info("Invalid MOE module, already disposed (" + module.getName() + ")");
-            return false;
-        }
-
-        ModuleWithDependenciesScope libraries = (ModuleWithDependenciesScope) module.getModuleWithLibrariesScope();
-        Collection<VirtualFile> roots = libraries.getRoots();
-
-        boolean coreFound = false;
-        boolean iosFound = false;
-        for (VirtualFile vf : roots) {
-            String name = vf.getName();
-            coreFound = coreFound ? coreFound : name.equals("moe-core.jar");
-            iosFound = iosFound ? iosFound : name.equals("moe-ios.jar");
-            if (coreFound & iosFound) {
-                return true;
-            }
-        }
-
-        return false;
-    }
+//    public static boolean isMoeJarsInModule(Module module) {
```

Comment:
Is there a reson, why it is only commented out and not deleted?

Reply:
Ah I forgot to delete those.